content/installation/java/configuration/YAML-properties-java.md

```diff
@@ -0,0 +1,203 @@
+    * **backups**: Specify the number of backup logs that the agent will create before Contrast cleans up the oldest file. A value of `0` means that no backups are created, and the log is truncated when it reaches its size cap. <br> Note: This property must be used with `agent.security_logger.roll_daily=false`; otherwise, Contrast continues to log daily and disregard this limit.
+
+    * **syslog**:
+      * **enable**: Set to `true` to enable Syslog logging.
+      * **ip**: Set the IP address of the Syslog server to which the agent should send messages.
+      * **port**: Set the port of the Syslog server to which the agent should send messages.
+      * **facility**: Set the facility code of the messages the agent sends to Syslog.
+      * **severity_exploited**: Set the log level of Exploited attacks. Value options are `ALERT`, `CRITICAL`, `ERROR`, `WARNING`, `NOTICE`, `INFO`, and `DEBUG`.
+      * **severity_blocked**: Set the log level of Blocked attacks. Value options are `ALERT`, `CRITICAL`, `ERROR`, `WARNING`, `NOTICE`, `INFO`, and `DEBUG`.
+      * **severity_probed**: Set the log level of Probed attacks. Value options are `ALERT`, `CRITICAL`, `ERROR`, `WARNING`, `NOTICE`, `INFO`, and `DEBUG`.
+
+
+<!-- ### Agent-Specific Properties
+
+That work will be done in CONTRAST-26166
+
+Words here...
+
+Properties formatted as list/grid -->
+
+
+### Inventory properties
+
+Use the properties in this section to control inventory features in the Java agent.
+
+* **inventory**:
+  * **enable**: Set to `false` to disable Inventory features in the agent.
+  * **library_dirs**: Define a list of directories where libraries are stored. Directories must be formatted as a semicolon-delimited list. <br> Example: `path1;path2;path3`
+  * **library_depth**: Set the maximum archive unpacking depth when analyzing libraries. <br> Example: `10`
+  * **prune_package_details**: Set to `false` to disable Inventory features in the agent. 
+  * **tags**: Apply a list of labels to libraries. Labels must be formatted as a comma-delimited list. <br> Example: `label1, label2, label3`
+
+### Contrast Assess properties
+
+Use the properties in this section to control Assess in the Java agent. The sampling settings allow you to control which requests the agent tracks and which it ignores. The rules setting allows you to control which Assess rules are disabled. 
+
+> **Note:** If you need a complete list of rules, use the **Support** widget in OpenDocs to contact Contrast's Customer Support team.  
+
+
+* **assess**:
+
+  * **enable**: Include this property to determine if the Assess feature should be enabled. If this property is not present, the decision is delegated to the Contrast UI. <br> Example: `true`
+  * **tags**: Apply a list of labels to vulnerabilities and preflight messages. Labels must be formatted as a comma-delimited list. Example: `label1, label2, label3`
+  * **stacktraces**: Value options are `ALL`, `SOME`, or `NONE`.
```

Comment:
This line needs to be removed. The java agent doesn't support `assess.stacktraces`.